dashboard/app/models/foorm/simple_survey_submission.rb

```diff
@@ -2,12 +2,13 @@
 #
 # Table name: foorm_simple_survey_submissions
 #
-#  id                  :integer          not null, primary key
-#  foorm_submission_id :integer          not null
-#  user_id             :integer
-#  misc_form_path      :string(255)
-#  created_at          :datetime         not null
-#  updated_at          :datetime         not null
+#  id                    :integer          not null, primary key
+#  foorm_submission_id   :integer          not null
+#  user_id               :integer
+#  simple_survey_form_id :bigint
```

Comment:
Mostly curious, why is this field a bigint?  It doesn't seem like we would have that many distinct surveys in the system?  (If anything, the submission id should be a big int.)

Reply:
Looks like we default to bigint now on IDs in new tables (I didn't specify anything about the ID in my migration, and it's bigint) -- we have had a couple near overflow situations over the past few years, so maybe it's just to avoid that generally?

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/db/migrate/20210429183952_create_foorm_simple_survey_forms.rb

Totally agree with what you're saying, but thought it'd be better to keep in sync with the primary key it is referencing.